docs/ImportantPythonFunctions.md

```diff
@@ -0,0 +1,180 @@
+
+# Save the TensorProto
+with open(os.path.join(os.path.dirname('tensor.pb'), 'wb') as f:
+    f.write(tensor.SerializeToString())
+
+# Load a TensorProto
+new_tensor = onnx.TensorProto()
+with open(os.path.join(os.path.dirname('tensor.pb'), 'rb') as f:
+    new_tensor.ParseFromString(f.read())
+print('After saving and loading, new TensorProto:\n{}'.format(new_tensor))
+```
+Runnable IPython notebooks:
+- [np_array_tensorproto.ipynb](https://github.com/onnx/onnx/tree/master/onnx/examples/np_array_tensorproto.ipynb)
+
+## Creating an ONNX Model Using Helper Functions
+```python
+import onnx
+from onnx import helper
+from onnx import AttributeProto, TensorProto, GraphProto
+
+
+# The protobuf definition can be found here:
+# https://github.com/onnx/onnx/blob/master/onnx/onnx.proto
+
+
+# Create one input (ValueInfoProto)
+X = helper.make_tensor_value_info('X', TensorProto.FLOAT, [1, 2])
+
+# Create one output (ValueInfoProto)
+Y = helper.make_tensor_value_info('X', TensorProto.FLOAT, [1, 2])
+
+# Create a node (NodeProto)
+node_def = helper.make_node(
+    'Pad', # node name
+    ['X'], # inputs
+    ['Y'], # outputs
+    mode='constant', # Attributes
+    value=1.5,
+    pads=[0, 1, 0, 1],
+)
+
+# Create the graph (GraphProto)
+graph_def = helper.make_graph(
+    [node_def],
+    "test-model",
+    [X],
+    [Y],
+)
+
+# Create the model (ModelProto)
+model_def = helper.make_model(graph_def, producer_name='onnx-example')
+
+print('The model is:\n{}'.format(model_def))
+onnx.checker.check_model(model_def)
+print('The model is checked!')
+```
+Runnable IPython notebooks:
+- [make_model.ipynb](https://github.com/onnx/onnx/tree/master/onnx/examples/make_model.ipynb)
+- [Protobufs.ipynb](https://github.com/onnx/onnx/tree/master/onnx/examples/Protobufs.ipynb)
+
+## Checking an ONNX Model
+```python
+import onnx
+import os
+
+# Preprocessing: load the ONNX model
+model_path = os.path.join(os.path.dirname('model.onnx')
+onnx_model = onnx.load(model_path)
+
+print('The model is:\n{}'.format(onnx_model))
+
+# Check the model
+onnx.checker.check_model(onnx_model)
+print('The model is checked!')
+```
+Runnable IPython notebooks:
+- [check_model.ipynb](https://github.com/onnx/onnx/tree/master/onnx/examples/check_model.ipynb)
+
+## Optimizing an ONNX Model
+```python
+import onnx
+import os
+from onnx import optimizer
+
+# Preprocessing: load the model contains two transposes.
+model_path = os.path.join('path/to/the/model')
+original_model = onnx.load(model_path)
+original_model_str = original_model.SerializeToString()
+
+print('The model before optimization:\n{}'.format(original_model))
+
+# A full list of supported optimization passes can be found here:
+# https://github.com/onnx/onnx/blob/master/onnx/optimizer.py#L21
+pass_list = ['fuse_consecutive_transposes']
+
+# Apply the optimization on the original serialized model
+optimized_model = onnx.load_from_string(optimizer.optimize(original_model_str, pass_list))
```

Comment:
should we have a wrapper function that hides to/from string? and also there should be a "no-brainer" pass list as default